Command/Message/example3.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "mega_rover_01",
+  "type": "mega_rover",
+  "time": "2019-06-07T08:39:40.064+09:00",
+  "command": "navi",
+  "waypoints": [
```

Comment:
the current geojson points allow the 3 coordinates. it has to be justified to create another system similar but not compatible.